pages/common/flutter.md

```diff
@@ -25,7 +25,7 @@
 
 - List or change Flutter channel:
 
-`flutter channel {{stable|beta|dev|master}}`
+`flutter channel {{stable|beta|dev|main}}`
```

Comment:
This shouldn't be changed, as it's the literal name of the release channel.
https://github.com/flutter/flutter/wiki/Flutter-build-release-channels